<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>350</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Gen 6</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButtonStationary6">
         <property name="text">
          <string>Stationary</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButtonEvent6">
         <property name="text">
          <string>Event</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Gen 7</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButtonStationary7">
         <property name="text">
          <string>Stationary</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButtonWild7">
         <property name="text">
          <string>Wild</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButtonEvent7">
         <property name="text">
          <string>Event</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="pushButtonID7">
         <property name="text">
          <string>TID/SID</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuStyle">
     <property name="title">
      <string>Style</string>
     </property>
     <addaction name="actionDark"/>
     <addaction name="actionLight"/>
    </widget>
    <widget class="QMenu" name="menuThreads">
     <property name="title">
      <string>CPU Threads</string>
     </property>
    </widget>
    <addaction name="menuThreads"/>
    <addaction name="actionProfiles"/>
    <addaction name="menuStyle"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionCalibrator7"/>
   </widget>
   <addaction name="menuSettings"/>
   <addaction name="menuTools"/>
  </widget>
  <action name="actionDark">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dark</string>
   </property>
  </action>
  <action name="actionLight">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Light</string>
   </property>
  </action>
  <action name="actionCalibrator7">
   <property name="text">
    <string>Gen 7 Calibrator</string>
   </property>
  </action>
  <action name="actionProfiles">
   <property name="text">
    <string>Profiles</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
